### ISSUE TITLE
Issues with card tracking during War Game

### ISSUE THREAD

**Issue by haydenreich** [DripDry/Zimmerer_CSCI2270_FinalProject#11]
In the war game, the same card can get played multiple times by either player, often within 1 or two rounds.  This happens both during war exchanges and during normal play. ie:

Player has 27 Cards left
Computer has 25 Cards left
Player plays:
9 of Hearts
Computer plays:
Queen of Clubs
Computer Wins!

=====Game Menu=====
1. Play a Round
2. Reset Game
3. Quit War
1

Player has 26 Cards left
Computer has 26 Cards left
Player plays:
Queen of Clubs
Computer plays:
7 of Diamonds
Player Wins!

**Comment by OmniBro** [comment]
That looks like an issue where the cards are added to the wrong side of the
queue.  Should be an easy fix

**Comment by haydenreich** [comment]
Good call, seems like that would work.  And great project, very ambitious and well executed
